src/main/java/lotto/Calculator.java

```diff
@@ -0,0 +1,28 @@
+package lotto;
+
+import java.math.BigDecimal;
+import java.util.Map;
+
+public class Calculator {
+    private static final BigDecimal PRICE_PER_LOTTO = BigDecimal.valueOf(1000);
+
+    public double calculateYield(BigDecimal purchaseAmount, BigDecimal prizeMoney) {
+        double earnedMoney = prizeMoney.subtract(purchaseAmount).doubleValue();
+        return 1 + (earnedMoney / purchaseAmount.doubleValue());
+    }
+
+    public int calculatePurchasedLottoNum(BigDecimal payAmount) {
+        return payAmount.divide(PRICE_PER_LOTTO).intValue();
+    }
+
+    public BigDecimal calculatePrizeMoney(Map<Integer, Integer> rankMap) {
+        BigDecimal prizeMoney = BigDecimal.ZERO;
+
+        for (Prize value : Prize.values()) {
+            prizeMoney = prizeMoney.add(
+                    BigDecimal.valueOf(rankMap.get(value.getWinningCount())).multiply(value.getPrizeMoney()));
```

Comment:
```suggestion
            BigDecimal prize = BigDecimal.valueOf(rankMap.get(value.getWinningCount())).multiply(value.getPrizeMoney());
            prizeMoney = prizeMoney.add(prize);
```
파라미터 안에 계산식이 들어가면 읽는데 막힘이 생겨요. 한 번 끊어주고 가면 좋을 것 같습니다!